www/docs/divisions/division.php

```diff
@@ -18,11 +22,17 @@
     $template = 'divisions/vote';
     $DATA->set_page_metadata($this_page, 'title', $data['division']['division_title']);
 
+    $data['main_vote_mp'] = $main_vote_mp;
```

Comment:
Not to do with this PR, but if we reversed these if statements and exited if there was no `$vote` or no `$data['division'` we'd have a lot less indenting.